Flick/SceneDelegate.swift

```diff
@@ -32,13 +34,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // TODO: Let's make another launch screen we show before navigating to either login or home
         window.rootViewController = navigationController
         window.makeKeyAndVisible()
-        guard let token = AccessToken.current, !token.isExpired else {
-            LoginManager().logOut()
-            UserDefaults.standard.removeObject(forKey: Constants.UserDefaults.authorizationToken)
-            UserDefaults.standard.removeObject(forKey: Constants.UserDefaults.userId)
```

Comment:
got rid of this since this only for FB ppl i think it's ok to just check if there's authorizationToken in userDefaults

Reply:
ooo k